### DISCUSSION TITLE
LineChart and AreaChart mix

### DISCUSSION THREAD

**Discussion by kayGweb** [Abhinandan-Kushwaha/react-native-gifted-charts#258] [Q&A] [answered]
Hello guys,
First, I enjoy my work with the library, which provides most of my current client's needs. However, I do have a question concerning the limitations or capabilities in its current form. We have a requirement that one of the lines in a line chart has an areaChart feel with gradient and opacities, and the other lines do not have the areaChart staying. Is that possible?? If so, how would I execute that?

Thanks,

**Comment by Abhinandan-Kushwaha** [comment, marked as answer]
Hi @kayGweb 👋
This is now possible in the latest version of the library. You can give one of the lines an areachart feel using the `areaChart1` `areaChart2` ... `areaChart5` props.
